tools/exporters/data_dictionary_builder.php

```diff
@@ -370,51 +373,6 @@ function enumizeOptions($options, $table, $name)
     return "enum($enum)";
 }
 
-/**
```

Comment:
these functions now exist directly in the database class as `pselectColWithIndexKey` and `pselectCol`